data/abilities.ts

```diff
@@ -1217,7 +1217,7 @@ export const Abilities: import('../sim/dex-abilities').AbilityDataTable = {
 					active.switchFlag = false;
 				}
 			}
-			target.switchFlag = true;
+			target.switchFlag = this.toID('emergencyexit');
```

Comment:
I think this is how I've seen it written in other places:
```suggestion
			target.switchFlag = 'emergencyexit' as ID;
```